lib/src/message_input.dart

```diff
@@ -258,7 +262,7 @@ class MessageInputState extends State<MessageInput> {
             });
           },
           style: widget.inputTextStyle ?? Theme.of(context).textTheme.bodyText2,
-          autofocus: false,
+          autofocus: widget.autofocus ?? false,
```

Comment:
Can you please set the default value in the constructor itself. We do not really need to handle the null case as the textField itself throws an assertion error.

Reply:
I'll merge it and add this change

Reply:
Sure thing. Changed in the last commit, please check.